scripts/js/lib/api/updateLinks.ts

```diff
@@ -169,25 +169,30 @@ export function relativizeLink(link: Link): Link | undefined {
     ["https://qiskit.org/documentation/stubs/", "/api/qiskit"],
     ["https://docs.quantum.ibm.com/", ""],
     ["https://docs.quantum-computing.ibm.com/", ""],
-    ["https://quantum.cloud.ibm.com/docs", "/docs"],
-    ["https://quantum.cloud.ibm.com/learning", "/learning"],
+    ["https://quantum.cloud.ibm.com/", ""], // ✅ New domain added
```

Comment:
I think we are missing the `/` here:

```suggestion
    ["https://quantum.cloud.ibm.com/", "/"],
```